docs/src/pages/css-in-js/advanced/advanced.md

```diff
@@ -247,6 +247,10 @@ function App() {
 export default App;
 ```
 
+### makeStyles
+
+Whenever you need to make use of the `useStyles` hook, you should instantiate a new instance by calling `makeStyles()` in each and every module. If you import an instance from another module, the CSS injection order could be broken. It may also only manifest when building in `production` mode, so be careful with that if you witness different behavior between `dev` and `production`.
```

Comment:
```suggestion
When using `useStyles`, you should instantiate a new instance by calling `makeStyles()` in each file. If you import an instance from another file, the CSS injection order can break and cause different behaviour in `dev` and `production`.
```
What do you think of this instead? I would get another contributors opinion too.

Reply:
That’s indeed way cleaner, thx!